### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by will-chow**
removed vs icon which was taking up to much of the page

### CODE REVIEW

**Comment by gewarren** [comment]
@will-chow Can you describe how you were viewing this page (mobile or not, browser type, screen resolution) that made the icon appear too large? I think a better option here is just to resize the icon. I'll suggest an edit.

File: docs/index.md [hunk 1]
```diff
@@ -13,25 +13,4 @@ manager: douge
...
-
-<ul class="panelContent cardsFTitle">
-    <li>
-        <a href="faq.md">
-        <div class="cardSize">
-            <div class="cardPadding">
-                <div class="card">
-                    <div class="cardImageOuter">
-                        <div class="cardImage">
-                            <img src="https://docs.microsoft.com/media/logos/logo_vs-ide.svg" alt="Visual Studio logo">
```
Comment (gewarren) [inline docs/index.md]: Instead of deleting this section, lets change line 26 to \

Comment (will-chow) [reply docs/index.md]: That's a great solution!

**Comment by gewarren** [comment]
I resized the icon in our private repo. The change should be live within a few minutes. Thanks for letting us know.

**Pull request closed without merging**